epics/ca.py

```diff
@@ -584,7 +591,10 @@ def _onGetEvent(args, **kws):
     if args.status != dbr.ECA_NORMAL:
         return
 
-    get_cache(name(args.chid))[args.usr] = memcopy(dbr.cast_args(args))
+    if dbr.isIronPython():
```

Comment:
This function call will be done on every `get` callback from libca. It should be just made into a boolean in `dbr.py`.